src/mongo/scripting/mozjs/jsstringwrapper.cpp

```diff
@@ -45,7 +45,8 @@ namespace mozjs {
 JSStringWrapper::JSStringWrapper(std::int32_t value) : _isSet(true) {
     auto formatted = fmt::format_int(value);
     _length = formatted.size();
-    strncpy(_buf, formatted.c_str(), sizeof(_buf));
+    memset(_buf, 0, sizeof(_buf));
```

Comment:
Instead of doing a `memset` here, why not just do `_buf[sizeof(buf)-1] = '\0'` after calling `strncpy`?

Reply:
Updated